File: src/main/java/no/nav/veilarboppfolging/controller/request/AktiverArbeidssokerData.java
package no.nav.veilarboppfolging.controller.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AktiverArbeidssokerData {
    Fnr fnr;
    Innsatsgruppe innsatsgruppe;

    @Data
    @NoArgsConstructor
    @AllArgsConstructor
    public static class Fnr {
        private String fnr;
    }
}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/request/AktiverArbeidssokerData.kt
package no.nav.veilarboppfolging.controller.request


data class AktiverArbeidssokerData(
    val fnr: Fnr,
    val innsatsgruppe: Innsatsgruppe,
) {
    data class Fnr(
        val fnr: String
    )
}
